### ISSUE TITLE
You can use DI-D stream API to animate the image character in realtime

### ISSUE THREAD

**Issue by Jaykef** [Shaunwei/RealChar#127]

**Comment by Jaykef** [comment]
yeah here: https://docs.d-id.com/reference/createtalkstream

I've built one using it:

![IMG_1421](https://github.com/Shaunwei/RealChar/assets/11355002/90837ed6-878f-43be-9b15-05350fa42945)

**Comment by adityabrahmankar** [comment]
Yeah! And they have an example repo too : https://github.com/de-id/live-streaming-demo

**Comment by ke-dev** [comment]
Streaming is too slow, animation is the fastest, like callannie.

**Comment by Jaykef** [comment]
> yeah here: https://docs.d-id.com/reference/createtalkstream
> 
> I've built one using it: AvaChat (a realtime AI chat demo with animated talking heads)

Try Demo Here: https://jaykef.github.io/AvaChat/
Repo: https://github.com/Jaykef/AvaChat

<img width="1710" alt="Screenshot 2023-09-05 at 23 14 55" src="https://github.com/Shaunwei/RealChar/assets/11355002/272e5e01-b0a8-4132-818d-4ea1a3e1b3cf">

**Comment by Jaykef** [comment]
> @Jaykef Do they have stream API?

yes, try demo here, I've added free api keys: https://jaykef.github.io/AvaChat/
